easybuild/easyblocks/n/namd.py

```diff
@@ -208,14 +208,14 @@ def test_step(self):
                 'testdir': os.path.join(self.cfg['start_dir'], self.namd_arch, 'src', 'alanin'),
                 'testopts': self.cfg['testopts'],
             }
-            out, ec = run_cmd(cmd, simple=False)
-            if ec == 0:
+            res = run_shell_cmd(cmd)
```

Comment:
isn't the exit code always 0 here since it would raise an exception otherwise?

Reply:
5a4e5f7